Zend/zend_vm_def.h

```diff
@@ -9808,6 +9808,44 @@ ZEND_VM_HANDLER(209, ZEND_INIT_PARENT_PROPERTY_HOOK_CALL, CONST, UNUSED|NUM, NUM
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(210, ZEND_FETCH_DEFAULT_ARG, UNUSED, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+
+	zend_function *called_func = EX(call)->func;
+
+	zend_string *reflection_class_name = ZSTR_INIT_LITERAL("ReflectionParameter", 0);
+	zend_class_entry *reflection_class = zend_fetch_class(reflection_class_name, ZEND_FETCH_CLASS_DEFAULT);
```

Comment:
Internal classes generally don't need to be fetched. You could just use the `reflection_class_ptr` variable.

Reply:
I don't understand how I can get a handle to this? It belongs to a different extension...

Reply:
Reflection is a required extension, it can never not be there. So it's fine to include its header.